conda/recipes/ucx-py/conda_build_config.yaml

```diff
@@ -1,4 +1,6 @@
 cython:
   - " >=0.29.14,<3.0.0a0"
+pynvml:
+  - " >=11.4.1"
```

Comment:
Offline Jordan suggested we include the `pynvml` version here. This seemed reasonable and gave this a try. Unfortunately it seems the version doesn't get picked up correctly.

<details>

```
$ git commit -v
[add_conda_recipe 27fab9f] Put `pynvml` version in `conda_build_config.yaml`
 2 files changed, 3 insertions(+), 1 deletion(-)
(base) jkirkham-mlt:ucx-py jkirkham$ conda render conda/recipes/ucx-py
/Users/jkirkham/miniconda/lib/python3.9/site-packages/conda_build/environ.py:438: UserWarning: The environment variable 'VERSION_SUFFIX' is undefined.
  warnings.warn(
/Users/jkirkham/miniconda/lib/python3.9/site-packages/conda_build/environ.py:438: UserWarning: The environment variable 'CC' is undefined.
  warnings.warn(
/Users/jkirkham/miniconda/lib/python3.9/site-packages/conda_build/environ.py:438: UserWarning: The environment variable 'CXX' is undefined.
  warnings.warn(
/Users/jkirkham/miniconda/lib/python3.9/site-packages/conda_build/environ.py:438: UserWarning: The environment variable 'CUDAHOSTCXX' is undefined.
  warnings.warn(
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): conda.anaconda.org:443
DEBUG:urllib3.connectionpool:https://conda.anaconda.org:443 "GET /t/ja-1d2faa65-06e2-47d9-80ad-b30203c92c09/conda-forge/osx-64 HTTP/1.1" 302 None
DEBUG:urllib3.connectionpool:https://conda.anaconda.org:443 "GET /conda-forge/osx-64/ HTTP/1.1" 200 None
Returning non-final recipe for ucx-py-0.26.00a-py39_gg27fab9f_14; one or more dependencies was unsatisfiable:
WARNING:conda_build.render:Returning non-final recipe for ucx-py-0.26.00a-py39_gg27fab9f_14; one or more dependencies was unsatisfiable:
Host: ucx
WARNING:conda_build.render:Host: ucx
--------------
Hash contents:
--------------
{'cython': ' >=0.29.14,<3.0.0a0',
 'pynvml': ' >=11.4.1',
 'target_platform': 'osx-64',
 'ucx': '1.12.1'}
----------
meta.yaml:
----------
package:
  name: ucx-py
  version: 0.26.00a
source:
  git_url: ../../..
build:
  number: '14'
  script:
    - /Users/jkirkham/miniconda/conda-bld/ucx-py_1651261743420/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python
      -m pip install . -vv
  script_env:
    - VERSION_SUFFIX
    - CC
    - CXX
    - CUDAHOSTCXX
  string: py39_gg27fab9f_14
requirements:
  host:
    - ucx 1.12.1.*
    - python 3.9.*
    - cython >=0.29.14,<3.0.0a0
    - pip
  run:
    - python >=3.9,<3.10.0a0
    - psutil
    - numpy
    - ucx
  run_constrained:
    - pynvml
test:
  imports:
    - ucp
  requires:
    - cudatoolkit 11.2.*
about:
  dev_url: https://github.com/rapidsai/ucx-py
  doc_url: https://ucx-py.readthedocs.io
  home: http://rapids.ai/
  license: BSD-3-Clause
  license_family: BSD
  license_file: ../../../LICENSE
  summary: Python bindings for UCX
extra:
  copy_test_source_files: true
  final: true
```

</details>

Reply:
Resolved in commit ( https://github.com/rapidsai/ucx-py/pull/843/commits/19ac6851bfdd564474b87fc93ab9a080328f4768 )

Reply:
Filed upstream issue ( https://github.com/conda/conda-build/issues/4446 ) about this behavior.